src/Effects/EffectLFO.h

```diff
@@ -41,6 +42,20 @@ class EffectLFO
         // current setup
         float samplerate_f;
         float buffersize_f;
+        float dt;
+
+        bool first_half;
+        float last_random;
+        
+        float FcAbs, K, norm, freq;
+
+        //biquad coefficients for lp filtering in noise-LFO
+        float a0 = 0.00017046738216391932;
+        float a1 = 0.00034093476432783864;
+        float a2 = 0.00017046738216391932;
+        float b1 = -1.9623638210277727;
+        float b2 =  0.9630456905564285;
+        float z1, z2;
```

Comment:
Perhaps I'm mistaken, but I don't see an initializer for these history values.

Reply:
fixed